contracts/0.4.24/nos/NodeOperatorsRegistry.sol

```diff
@@ -101,10 +101,16 @@ contract NodeOperatorsRegistry is INodeOperatorsRegistry, IsContract, AragonApp
         validAddress(_rewardAddress)
         returns (uint256 id)
     {
-        id = totalOperatorsCount++;
+        bytes32 totalOperatorsCountPosition = TOTAL_OPERATORS_COUNT_VALUE_POSITION;
```

Comment:
No need to cache `TOTAL_OPERATORS_COUNT_VALUE_POSITION` here: since it's a constant, reading it doesn't produce and `SLOAD`s.

Reply:
Agree, fixed.